### ISSUE TITLE
devtools::check fails on documentation issue

### ISSUE THREAD

**Issue by Kdreval** [morinlab/GAMBLR#169]
The `devtools::check()` returns a series of warnings indicating problems with package build. This one refers to the documentation:

```
❯ checking for unstated dependencies in examples ... WARNING
  Warning: parse error in file 'lines':
  35: unexpected symbol
  217:                                   custom_colours = c("#onecolour, "#anothercolour", "athirdcolour"),
  218:                                   classification_column
                                         ^
```

**Comment by Kdreval** [comment]
Also this is related to the documentation:

```
❯ checking Rd cross-references ... WARNING
  Missing link or links in documentation object 'FtestCNV.Rd':
    ‘3’ ‘"GRAPH"’
  
  Missing link or links in documentation object 'calculate_pga.Rd':
    ‘0..1’
  
  Missing link or links in documentation object 'prettyChromoplot.Rd':
    ‘0-1’
  
  See section 'Cross-references' in the 'Writing R Extensions' manual.
```

**Comment by Kdreval** [comment]
And this:
```
❯ checking for unstated dependencies in examples ... WARNING
  Warning: parse error in file 'lines':
  6: unexpected symbol
  1562: MYC_expr = get_gene_expression(hugo_symbols = c("MYC"), join_with = "mrna")
  1563: Read full
             ^
```

**Comment by Kdreval** [comment]
The list extended to other issues as well. Apparently, devtools reported these syntax issues one at a time. All were resolved in #177
